### DISCUSSION TITLE
How to discuss not publicly (within maintainers group)

### DISCUSSION THREAD

**Discussion by wernerfred** [docker-mailserver/docker-mailserver#3179] [General]
I wanted to share some governance/structural findings for this project and was looking to start a thread in the maintainers team like we did earlier since bringing the repo in this organization. But the teams discussions are disabled and deprecated in favor of the public organizational discussion.

So my question: how can we share some insights first in a smaller audience before opening a public issue/discussion? Not only for my case now, but also for people wanting to share maybe some security concerns? GitLab has a feature to open issues as confidential, do we have some comparable feature available here as well?

@georglauterbach @casperklein @polarathene

**Comment by casperklein** [comment]
See [here](https://github.com/docker-mailserver/docker-mailserver/issues/3148).

**Reply by wernerfred** [reply]
Shame on me 😶
